src/doc/reference.md

```diff
@@ -4062,7 +4084,7 @@ the guarantee that these issues are never caused by safe code.
 * Breaking the [pointer aliasing
   rules](http://llvm.org/docs/LangRef.html#pointer-aliasing-rules)
   with raw pointers (a subset of the rules used by C)
-* `&mut T` and `&T` follow LLVM’s scoped [noalias] model, except if the `&T`
+* `&mut` and `&` follow LLVM’s scoped [noalias] model, except if the `&T`
```

Comment:
Yeah I don't know what happened, I think I had the tab open too long and someone else landed a commit to the reference. I'll rebase to get rid of it. I only meant to edit the Macros section.